packages/react-accessible-tooltip/package.json

```diff
@@ -28,53 +22,50 @@
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-2": "^6.24.1",
     "classnames": "^2.2.5",
-    "enzyme": "^3.1.1",
-    "enzyme-adapter-react-16": "^1.0.4",
-    "enzyme-to-json": "^3.2.2",
-    "eslint": "^4.11.0",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-to-json": "^3.3.1",
+    "eslint": "^4.18.2",
     "eslint-config-airbnb": "^16.1.0",
-    "eslint-config-prettier": "^2.7.0",
-    "eslint-plugin-flowtype": "^2.39.1",
-    "eslint-plugin-flowtype-errors": "^3.3.5",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jest": "^21.3.2",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-flowtype": "^2.46.1",
+    "eslint-plugin-flowtype-errors": "^3.5.1",
+    "eslint-plugin-import": "^2.7.0",
```

Comment:
Is this a deliberate downgrade?

Reply:
Great spotting: the versions of `eslint-plugin-import`, `eslint-plugin-jsx-a11y` and `eslint-plugin-react` are all actually determined by the peerDependencies specified by `eslint-config-airbnb`. As it turns out, I was a little bit eager in upgrading `eslint-plugin-import` prior to this PR.